src/Shout/SHRBTextStyler.class.st

```diff
@@ -1028,6 +1028,8 @@ SHRBTextStyler >> resolveStyleFor: aVariableNode [
 	"here we should add support for #classVar"
 	aVariableNode isClassVariable ifTrue: [ ^#globalVar].
 	aVariableNode isInstanceVariable ifTrue: [ ^#instVar]. 
+	"for now we use temp variable here, we could now color them with their own color"
+	aVariableNode isWorkspaceVariable ifTrue: [ ^#tempVar]. 
```

Comment:
Now we can refactor it into "aVariableNode styleSymbol" and use it directly in senders of #resolveStyleFor: